app/views/compute_resources_vms/show/_gce.html.erb

```diff
@@ -5,28 +5,16 @@
       <tr><th colspan="2"><%=_("Properties") %></th></tr>
     </thead>
     <tbody>
+      <%= prop :state %>
+      <%= prop :pretty_machine_type, "Machine Type" %>
+      <%= prop :zone_name, "Zone" %>
       <tr>
-        <td>State</td>
-        <td><%= @vm.status.downcase %></td>
+        <td>Created</td>
+        <td><%= Time.parse(@vm.creation_timestamp).utc %></td>
```

Comment:
Should this use `date_time_relative_value` helper for consistency with other time fields?